cplusplus/include/vips/VImage8.h

```diff
@@ -352,9 +352,10 @@ class VImage : VObject
 	}
 
 	void 
-	set( const char *field, int *value, int n )
```

Comment:
How about supporting both? We don't need to remove the old setters -- overloading will pick the right one at compile-time.

Reply:
Good idea! I'll change my PR.

Reply:
OK, done!